agent/providers/api.go

```diff
@@ -8,8 +8,10 @@ import (
 	agentapi "github.com/synadia-io/nex/internal/agent-api"
 )
 
-// NexExecutionProviderELF Executable Linkable Format execution provider
-const NexExecutionProviderELF = "elf"
+// TODO: change this to "native" from "elf" when appropriate
+
+// NexExecutionProviderNative Executable Linkable Format execution provider
+const NexExecutionProviderNative = "elf"
```

Comment:
do you want to change the constant to "native" or something other than elf??

Reply:
yeah, at some point it'll be `native`. I just didn't want to make that change in this PR since it'll bust things